blueprints/initializer-test/mocha-rfc-232-files/__root__/__testType__/__path__/__name__-test.js

```diff
@@ -0,0 +1,32 @@
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+import { setupTest } from 'ember-mocha';
+import Application from '@ember/application';
+import { initialize } from '<%= modulePrefix %>/initializers/<%= dasherizedModuleName %>';
+<% if (destroyAppExists) { %>import destroyApp from '../../helpers/destroy-app';<% } else { %>import { run } from '@ember/runloop';<% } %>
+
+describe('<%= friendlyTestName %>', function() {
+  let hooks = setupTest();
```

Comment:
do we need this? we don't seem to be using anything from the test context 🤔

Reply:
hmm, looks like the QUnit variant is doing the same thing. @rwjblue do you remember why we use it there?

Reply:
Yes, I made it do the same as the QUnit one for now, but also wondered about this. See my note here: https://github.com/emberjs/ember.js/pull/17412

Reply:
@rwjblue do you? 👉 https://github.com/emberjs/ember.js/pull/17411#discussion_r244282578 
Would love to finish this and #17412, to update the TS blueprints as well! 😉

Reply:
No, seems like we can/should remove from both mocha and qunit. Sorry for the delay here.

Reply:
Removed `setupTest()`! Ready for final review.

Reply:
Here for QUnit again: https://github.com/emberjs/ember.js/pull/17713